### ISSUE TITLE
function not implemented (tar) *easy fix!*

### ISSUE THREAD

**Issue by suresttexas00** [benbaptist/minecraft-wrapper#77]
I have tar installed and it is in my system Path.  I get the same message:

```
[23:50:40] [Server thread/INFO]: Saving...
[23:50:40] [Server thread/INFO]: Saved the world
[23:50:40] [Server thread/INFO]: Turned off world auto-saving
C:\Program Files (x86)\GnuWin32\bin\tar.exe
tar: Cannot open /Users/JRUSSELL/Desktop/vanilla/backups//backup-2015-01-20_23:5
0:40.tar: Function not implemented
tar: Error is not recoverable: exiting now
[23:50:41] [Server thread/INFO]: Turned on world auto-saving
```

**Comment by suresttexas00** [comment]
I'll get that extra / out of there.. but I don't think that is the issue.  I am using 

" /Users/JRUSSELL/Desktop/vanilla/backups/" in the wrapper.properties (of course, I am removing the trailing slash and will see if I get same results -pretty sure I will.

**Comment by suresttexas00** [comment]
Yep.. same thing.

**Comment by suresttexas00** [comment]
Also tried it this way:

`C:\Users\JRUSSELL\Desktop\vanilla\backups`

```
[00:21:26] [Server thread/INFO]: Saving...
[00:21:26] [Server thread/INFO]: Saved the world
[00:21:26] [Server thread/INFO]: Turned off world auto-saving
C:\Program Files (x86)\GnuWin32\bin\tar.exe
tar: Cannot open C:\Users\JRUSSELL\Desktop\vanilla\backups/backup-2015-01-21_00:
21:26.tar: Function not implemented
tar: Error is not recoverable: exiting now
[00:21:27] [Server thread/INFO]: Turned on world auto-saving
```

**Comment by suresttexas00** [comment]
I'm not having any luck running tar from the command prompt either.  It is installed but I get errors like:
alltheworld is the world folder and backup is the file I want to create (for simplicity, right in the vanilla folder).

```
C:\Users\JRUSSELL\Desktop\vanilla>tar -cfv backup alltheworld
tar: Cannot add file backup: No such file or directory
tar: Error exit delayed from previous errors

C:\Users\JRUSSELL\Desktop\vanilla>tar -c backup alltheworld
tar: Cannot open -: Invalid argument
tar: Error is not recoverable: exiting now

C:\Users\JRUSSELL\Desktop\vanilla>tar -c backup
tar: Cannot open -: Invalid argument
tar: Error is not recoverable: exiting now

C:\Users\JRUSSELL\Desktop\vanilla>tar -c backup.tar
tar: Cannot open -: Invalid argument
tar: Error is not recoverable: exiting now
```

**Comment by benbaptist** [comment]
Try doing it without the -. 

`tar cfpv backup.tar alltheworld`

I think it's also important to actually add the .tar to the end - otherwise it might bug out.

**Comment by suresttexas00** [comment]
OK! That worked.. now how do I get that to wrapper?

The command I need (that works) is:
`tar cfpv /Users/JRUSSELL/Desktop/vanilla/backu
ps/backup.tar \Users\JRUSSELL\Desktop\vanilla\AllTheWorld`

**Comment by suresttexas00** [comment]
I will see if taking the dash out of the source works

**Comment by suresttexas00** [comment]
I have narrowed the error as being something caused by the backup file name.  When i change the file name to just Backup (eliminate the timestamp), it works.

**Comment by suresttexas00** [comment]
Its the Colons!  You can't use those in windows file names.  May I suggest replacing those two colons with periods?
Works... 
![image](https://cloud.githubusercontent.com/assets/10001762/5849609/d80ee5d2-a1b5-11e4-92ac-bafff90de165.png)

**Comment by benbaptist** [comment]
I'm not at my computer right now, but I can fix as soon as I get back!

**Comment by benbaptist** [comment]
I just released a stable 0.7.5 build, and will probably do a 0.7.6 shortly to iron out some of the bugs and whatnot.

**Comment by suresttexas00** [comment]
Created the pull request to fix.  Code works great.
